MATH2269_final_project.Rmd

```diff
@@ -672,102 +697,175 @@ if(hasnt_run(trial_type)){
 
 ```
 
-
-```{r Regression Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "", fig.width=10}
+```{r Regression Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "The regression parameter model diagnostics for trial model_001_gamma_gamma_c3_a500_b500_t3.", fig.width=10}
 
 view_regression_diagnostics_images(trial_type)
 
 ```
 
-```{r Prediction Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "", fig.width = 10}
+In **beta[3]** trace plot we can see the start of the chains isolate around a higher mean than where they converge. With some of the density plots not overlapping either possibly a longer burn in could be used. This will allow the chains a longer period to converge before recording the model outputs.
+
+Thinning steps will also be increased to attempt to improve the last two parameters with autocorrelation left in them (**tau** and **??**).
+
+```{r Prediction Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "A sample of the prediction model diagnostics for trial model_001_gamma_gamma_c3_a500_b500_t3.", fig.width = 10}
 
 view_prediction_diagnostics_images(trial_type)
 
 ```
 
+```{r Posterior Images, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "The posteriod distribution for trial model_001_gamma_gamma_c3_a500_b500_t3.", fig.width = 10}
+
+view_posterior_images(trial_type)
+
+```
+In the posterior distributions we can also see that the distributions aren't smooth indicating that there is not enough data produced by the iterations of the MCMC model.
+
+
+```{r Summary Table}
+
+view_summary_table(trial_type) %>% format_table(p_caption="", text_size = 7)
+
+```
 
 
-```{r Posterior Images, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "", fig.width = 10}
+#### model_002_gamma_gamma_c3_a500_b1000_t5
+
+```{r}
+
+trial_type <- "model_002_gamma_gamma_c3_a500_b1000_t5"
+
+# Parameters to use
+use_parameters <- list(number_chains = 3, 
```

Comment:
This has been removed in https://github.com/ellenitoumpas/bayesian-final-project/pull/23, which I've merged, so you'll likely get conflicts anywhere that use_parameters is specified.